src/sentry/static/sentry/app/views/organizationDashboard/projectCard.jsx

```diff
@@ -49,6 +49,12 @@ class ProjectCard extends React.Component {
   }
 }
 
+const StyledLink = styled(Link)`
```

Comment:
Can we use `styles/overflowEllipsis` here? (Theres' also `<TextOverflow>` but that adds an extra container :/)